openhtf/core/test_record.py

```diff
@@ -72,7 +72,7 @@ class PhaseRecord(  # pylint: disable=no-init
     mutablerecords.Record(
         'PhaseRecord', ['name', 'codeinfo'],
         {'measurements': None,
-         'start_time_millis': None, 'end_time_millis': None,
+         'start_time_millis': int, 'end_time_millis': None,
```

Comment:
I'm saying that OpenHTF is historically un-opinionated about where and how you use it (and its parts).